src/main/java/com/zufar/onlinestore/payment/service/PaymentEventProcessor.java

```diff
@@ -0,0 +1,28 @@
+package com.zufar.onlinestore.payment.service;
+
+import com.stripe.exception.SignatureVerificationException;
+import com.stripe.model.Event;
+import com.stripe.model.PaymentIntent;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+@Slf4j
+@RequiredArgsConstructor
+@Service
+public class PaymentEventProcessor {
```

Comment:
add java doc please
it helps to understand this class for people who have no idea about stripe